src/test/java/org/traccar/ProtocolTest.java

```diff
@@ -11,19 +11,14 @@
 import io.netty.handler.codec.http.HttpResponseStatus;
 import io.netty.handler.codec.http.HttpVersion;
 import org.traccar.helper.DataConverter;
-import org.traccar.model.CellTower;
-import org.traccar.model.Command;
-import org.traccar.model.Position;
-import org.traccar.model.WifiAccessPoint;
+import org.traccar.model.*;
```

Comment:
We don't do star imports. And also let's separate all the test refactoring in a separate PR as well.